html/controls/operator.js

```diff
@@ -1089,6 +1089,214 @@ function createJamDialog() {
 	});
 }
 
+function createTimeoutDialog() {
+	var lastPeriodRegistered = 0;
+	var lastPeriodListed = 0;
+	var firstJamListed = [0];
+	var lastJamListed = [0];
+	var periodDropdownTemplate = $('<select>').attr('id', 'PeriodDropdown')
+		.append($('<option>').attr('value', 0).text('P0'));
+	var jamDropdownTemplate = [$('<select>').attr('id', 'JamDropdown').attr('period', 0)
+		.append($('<option>').attr('value', 0).text('J0'))];
+	var typeDropdownTemplate = $('<select>').attr('id', 'TypeDropdown')
+		.append($('<option>').attr('value', '.false').text('No type'))
+		.append($('<option>').attr('value', 'O.false').text('Off. Timeout'))
+		.append($('<option>').attr('value', '1.false').text('Team TO left'))
+		.append($('<option>').attr('value', '2.false').text('Team TO right'))
+		.append($('<option>').attr('value', '1.true').text('Off. Review left'))
+		.append($('<option>').attr('value', '2.true').text('Off. Review right'));
+	
+	var dialog = $("<div>").addClass("NumberDialog");
+	var table = $("<table>").appendTo(dialog);
+	var headers = $("<tr><td/><td/><td/><td/><td/><td/><td/></tr>").appendTo(table);
+	$("<a>").text("Period").addClass("Title")
+		.appendTo(headers.find("td:eq(0)").addClass("Title"));
+	$("<a>").text("After Jam").addClass("Title")
+		.appendTo(headers.find("td:eq(1)").addClass("Title"));
+	$("<a>").text("Duration").addClass("Title")
+		.appendTo(headers.find("td:eq(2)").addClass("Title"));
+	$("<a>").text("Period Clock").addClass("Title")
+	.appendTo(headers.find("td:eq(3)").addClass("Title"));
+	$("<a>").text("Type").addClass("Title")
```

Comment:
The retained column could do with a heading, it's a pixel wide until there's an OR currently.